base/compiler/optimize.jl

```diff
@@ -462,7 +466,7 @@ function finish(interp::AbstractInterpreter, opt::OptimizationState,
                 @assert isconstType(result)
                 analyzed = ConstAPI(result.parameters[1])
             end
-            force_noinline || (src.inlineable = true)
+            force_noinline || (src.inlining_cost = 0x0010)
```

Comment:
It would be better if we normalize the operations.
```suggestion
            force_noinline || set_inlineable!(src, true)
```